genpack.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/sh
+
+# generate `.tar.gz` packages from git repos
+
+# set PACKAGEURL
+set_pack_url() {
+	case "$1" in
+	tunasync)	PACKAGEURL=https://github.com/sjtug/tunasync.git ;;
```

Comment:
Thus, there are three options
1. Keep using git in genpack script
2. Use wget instead in genpack script
3. Scripts suck, use `RUN wget` in Dockerfile instead
Which one should we use?